docs/_scripts/download_after_sync.sh

```diff
@@ -3,8 +3,7 @@
 CONTAINER_TAG='manifest_a'
 
 echo "Setting REGISTRY_PATH, which can be used directly with the Docker Client."
-export REGISTRY_PATH=$(http $BASE_ADDR$DISTRIBUTION_HREF | jq -r '.registry_path')
-
-echo "Next we pull the image from pulp and run it."
+REGISTRY_PATH=$(http $BASE_ADDR$DISTRIBUTION_HREF | jq -r '.registry_path')
+echo "Next we pull the image from pulp"
 echo "$REGISTRY_PATH:$CONTAINER_TAG"
-sudo docker run $REGISTRY_PATH:$CONTAINER_TAG
+sudo docker pull $REGISTRY_PATH:$CONTAINER_TAG
```

Comment:
i needed to change run for pull, because the currently build images cannot be run and fail with the following error:
`Error response from daemon: No command specified`
The image is so small that it does not contain any command to be executed
I submitted the PR here https://github.com/pulp/pulp-fixtures/pull/204 but then we will need to rebuild the images and push to the docker hub

Reply:
I'm trying to run that script but I get:
```
**********Create and Push Manifest Lists*******
docker manifest create is only supported on a Docker cli with experimental cli features enabled
docker manifest push is only supported on a Docker cli with experimental cli features enabled
docker manifest create is only supported on a Docker cli with experimental cli features enabled
docker manifest push is only supported on a Docker cli with experimental cli features enabled
docker manifest create is only supported on a Docker cli with experimental cli features enabled
docker manifest push is only supported on a Docker cli with experimental cli features enabled
docker manifest create is only supported on a Docker cli with experimental cli features enabled
docker manifest push is only supported on a Docker cli with experimental cli features enabled
```
even putting "experimental": true on daemon.json didn't fix it for me

Reply:
I found the issue, I was editing the config on my HOME, but the script uses root